sympy/series/limits.py

```diff
@@ -124,6 +124,11 @@ def doit(self, **hints):
         from sympy.series.limitseq import limit_seq
 
         e, z, z0, dir = self.args
+        
+        if type(e) is exp:
```

Comment:
At first ,I tried this : 
`isinstance(type(e),exp)` but this is returning False.In my example
 `e = ex = exp(x*a)` and `a = symbols("a",zero =True)`.